examples/nidcpower/measure-record.py

```diff
@@ -166,12 +172,27 @@ def on_close(event):
             fetch_multiple_response = client.FetchMultiple(nidcpower_types.FetchMultipleRequest(
                 vi = vi,
                 channel_name = channels,
-                timeout = get_measure_record_delta_time.attribute_value,
+                timeout = 5,
                 count = record_length
             ))
             CheckForError(vi, fetch_multiple_response.status)
-            y_axis = fetch_multiple_response.voltage_measurements
-
+            
+            # Append the fetched values in the buffer.
+            y_axis.extend(fetch_multiple_response.voltage_measurements)
+            y_axis = y_axis[record_length:]
+            
+            # Updating the precision of the fetched values.
+            y_axis_new = []
+            for value in y_axis:
+                if (value < voltage_level):
+                    y_axis_new.append(math.floor(value*100)/100)
+                else:
+                    y_axis_new.append(math.ceil(value*100)/100)
+
+            # Plotting
+            y_axis = y_axis_new
+            x_axis = np.arange(start=x_start, stop=x_start + record_length * buffer_multiplier, step=1)
+            x_start = x_start + record_length
```

Comment:
This example is continuous measure record, so the above changes are made just to output a running x-axis in the plot. We are extending the buffer everytime new record is being fetched.